### DISCUSSION TITLE
Why wont my rich presence appear on mobile?

### DISCUSSION THREAD

**Discussion by liuu-xinn** [maximmax42/Discord-CustomRP#244] [Q&A]
Works perfectly fine within my PC client. I read it may be a priority issue, so I closed the PC client and rebooted the mobile client. It still won't appear - works fine on mobile for others I know who redirected me to this for help?

Using IOS

**Comment by liuu-xinn** [comment]
Also noting that I have read the FAQ and attempted all troubleshooting, the change in pipe does nothing, and share activity is enabled.

**Comment by maximmax42** [comment]
Does the app say "Updating presence..." or "Connected" in the bottom right of the app's window? If first, check your image links.
